tests/ui/conftest.py

```diff
@@ -1,11 +1,24 @@
 import datetime
+import json
 import os
 import urlparse
 
-from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
 import jwt
 import pytest
 import requests
+from django.core.management import call_command
+from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
+from olympia.amo.tests import create_switch
+from pytest_django import live_server_helper
+
+
+@pytest.fixture
+def local_base_url(live_server):
```

Comment:
How did you override ```base_url``` when it is function scoped? Everytime I tried it would say I was trying to use a session scope fixture in a function scope.